VirtoCommerce.WebhooksModule.Data/Repositories/IWebhookRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Linq;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.WebhooksModule.Data.Models;
+
+namespace VirtoCommerce.WebhooksModule.Data.Repositories
+{
+    public interface IWebHookRepository : IRepository
+    {
+        IQueryable<WebHookEntity> WebHooks { get; }
+        IQueryable<WebHookEventEntity> WebHookEvents { get; }
+        IQueryable<WebHookFeedEntryEntity> WebHookFeedEntries { get; }
+    }
```

Comment:
Please make `GetWebHooksByIds` and `DeleteWebHooksByIds` methods in repository and use them in services.